variants/neutralarmies/neutralarmies.go

```diff
@@ -0,0 +1,90 @@
+package neutralarmies
+
+import (
+	"time"
+
+	"github.com/zond/godip"
+	"github.com/zond/godip/orders"
+	"github.com/zond/godip/state"
+	"github.com/zond/godip/variants/classical"
+	"github.com/zond/godip/variants/classical/start"
+	"github.com/zond/godip/variants/common"
+	"github.com/zond/godip/variants/hundred"
+)
+
+func NeutralOrders(state state.State) (ret map[godip.Province]godip.Adjudicator) {
+	ret = map[godip.Province]godip.Adjudicator{}
+	if state.Phase().Type() == godip.Movement {
+
```

Comment:
What's this?

Reply:
Ah, this is when I thought Go didn't support switch statements.

Reply:
This is now fixed in #40